client/components/JestReact.test.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, screen} from '@testing-library/react';
```

Comment:
Missing whitespace after screen. Were these files linted? We can add that task to circleCI as well.

Reply:
Good eye, I did not see the lint on review because it's hard for me to see the red. I will get into the habit of running the lint command before each commit.